streams/src/main/java/org/apache/kafka/streams/state/internals/LRUCacheEntry.java

```
@@ -63,6 +63,10 @@ public long timestamp() {
         return timestamp;
     }
 
+    public void setTimestamp(final long timestamp) {
+        this.timestamp = timestamp;
```

Should we throw an unsupported exception here?